app/assets/javascripts/foreman_theme_satellite/locale/bn/foreman_theme_satellite.js

```diff
@@ -0,0 +1,124 @@
+ locales['foreman_theme_satellite'] = locales['foreman_theme_satellite'] || {}; locales['foreman_theme_satellite']['bn'] = {
```

Comment:
there is no `locale/bn/*.po` as there are no strings in that translation yet, but `po_to_json` still creates this file :confused:

Reply:
mhh, now it didn't. whatever.